actions/check_consistency.meta.yaml

```diff
@@ -0,0 +1,37 @@
+---
+name: "check_consistency"
+runner_type: "python-script"
+description: "Check that the device's configuration is consistent with the 'golden' config in a Git repository"
+enabled: true
+entry_point: "check_consistency.py"
+parameters:
+    hostname:
+        type: "string"
+        description: "The hostname of the device to connect to. Driver must be specified if hostname is not in configuration. Hostname without FQDN can be given if device is in configuration."
+        required: true
+        position: 0
```

Comment:
Python actions don't need positions. If it's a python script then, yes. The runner type being python-script can be confusing.